Prepare documentation for translation (#862)
ManimCommunity/manim: docs/source/contributing/internationalization.rst

====================
Internationalization
====================

If you want to help with translating the docs:

#. Make sure you have a `Transifex <https://www.transifex.com/>`_ account. Transifex is a tool for colaborative translation and allows the Manim team to review translations.
#. Open the `Manim Transifex project page <https://www.transifex.com/manim-community/manim-community-documentation>`_ .
#. Click on ``Join Team``.
#. Select the language you want to help with.

Congratulations! You are now a Translator for the Manim project!

> You have access to the projects you joined just on the left of your profile picture

Reviewer: ```suggestion
You'll see the projects you've joined just on the left of your profile picture
```

on the top bar.

.. image:: /_static/select_transifex_repo.png
   :align: center
   :width: 200

Now go to the
`languages list <https://www.transifex.com/manim-community/manim-community-documentation/languages/>`_
and select a language you want to help with. You should see a blue button reading

> `Translate`; click it to get started! You will see a list of

Reviewer: ```suggestion
`Translate`; click on it to get started! You will see a list of
```
